app/src/main/res/values-cn/strings.xml

```diff
@@ -0,0 +1,74 @@
+<resources>
+    <string name="app_name">Blue Line Console</string>
+    <string name="loading_main_caption">正在刷新数据库......</string>
+    <string name="loading_detail_for_command_search">安装后首次启动,或是更改语言设定后首次启动，\n刷新可能需要一段时间才能完成（有时几十秒）。</string>
+    <string name="formatSearchQueryOnEngine">在 %2$s 上搜索 \"%1$s\"</string>
+    <string name="formatStaticPageEntry">访问 %1$s</string>
+    <string name="displayedFullVersionString">版本 %1$s</string>
+    <string name="preference_title_for_header_and_footer">配置</string>
+    <string name="title_for_notification">通知</string>
+    <string name="notFoundOpeningApplicationWithClass">\"%1$s\" 不是一个可启动的包。</string>
+    <string name="calculator_precision_format">计算精度: %1$s</string>
+    <string name="calculator_precision_no_error">无差错</string>
+    <string name="calculator_precision_rounding">进位舍入</string>
+    <string name="calculator_error_division_by_zero">除以 0</string>
+    <string name="summary_text_for_preferences_of_this_app">Blue Line Console 选项</string>
+    <string name="notification_title_on_startup">关于“help” 命令</string>
+    <string name="notification_detail_on_startup_pre">要显示用法，输入</string>
+    <string name="notification_detail_on_startup_main" translatable="false">help</string>
```

Comment:
```suggestion
```
Can you remove this line as untranslatable strings in non-default language is duplicated code, and is reported as error in Android Studio?